compiler-core/src/package_interface/snapshots/gleam_core__package_interface__tests__generic_function.snap

```diff
@@ -0,0 +1,71 @@
+---
+source: compiler-core/src/package_interface/tests.rs
+expression: "\npub type Bar(a) { Bar }\n@deprecated(\"deprecation message\")\npub fn main() { Bar }\n"
+---
+{
+  "name": "my_package",
+  "package_version": {
```

Comment:
This should be `"version": "11.10.9-1foo"`. I think the default encoder for a version uses that format so you could try that.